### DISCUSSION TITLE
Compatibility with ARP

### DISCUSSION THREAD

**Discussion by mariuszparys** [Spellhold-Studios/Infinity-Sounds#11] [Questions]
Is Infinity Sound mod compatible with Almateria Restoration Pack (Restored Sounds)?

**Comment by skellytz** [comment]
ARP is a very good mod, but you must understand its approach, which can be summarized as: if a creature has this in select common, then assume it needs this in morale, damage, dying and other select common slots. This is done with SAY commands with blank text strings, e.g., SAY MORALE ~~ [SOUND]. So, on one hand, this restores sounds and ensures consistency between creatures. On the other hand, the sounds will have no subtitles and all the subtitles already associated with these sounds will be needlessly removed, even if a creature already had a valid string reference in that slot.

Infinity Sounds uses a more careful approach. It restores all unused or incorrectly assigned sounds (morale, battle cry, damage, dying, select, hostile, etc.) to verified creatures, but attempts to:
- use existing text strings with these sounds if they are present
- add sounds to existing text strings if they have only text but no sound
- add new subtitles to the game if the text string isn't available (currently English only, around 100 subtitles added)

As you can see both mods try to restore sounds, but they have different approaches. If you really want to combine these two methods, install ARP earlier than IS. We haven't tested the compatibility; there may be some duplicate, mismatched, or missing sounds with both mods installed.
